src/main/java/lotto/domain/LottoNumbers.java

```diff
@@ -0,0 +1,56 @@
+package lotto.domain;
+
+import java.util.Collections;
+import java.util.List;
+
+public class LottoNumbers {
+
+    private static final int LOTTO_NUMBERS_SIZE = 6;
+    private static final String DISTINCT_LOTTO_NUMBER_EXCEPTION_MESSAGE_FORMAT = "중복 된 로또 숫자가 존재합니다. lottoNumbers: %s";
+    private static final String INVALID_LIST_SIZE_EXCEPTION_MESSAGE_FORMAT = "로또 숫자의 개수가 6개가 아닙니다. lottoNumbers: %s";
+
+    private final List<LottoNumber> lottoNumbers;
+
+    private LottoNumbers(List<LottoNumber> lottoNumbers) {
+        validateSize(lottoNumbers);
+        validateDistinct(lottoNumbers);
+        this.lottoNumbers = Collections.unmodifiableList(lottoNumbers);
+    }
+
+    private void validateSize(List<LottoNumber> lottoNumbers) {
+        if (lottoNumbers.size() != LOTTO_NUMBERS_SIZE) {
+            throw new IllegalArgumentException(String.format(INVALID_LIST_SIZE_EXCEPTION_MESSAGE_FORMAT, lottoNumbers.toString()));
```

Comment:
```suggestion
            throw new IllegalArgumentException(String.format(INVALID_LIST_SIZE_EXCEPTION_MESSAGE_FORMAT, lottoNumbers));
```

toString()을 호출하지 않아도 객체가 문자화되면 toString()이 호출됩니다. :)